### DISCUSSION TITLE
Display: Number Formatting

### DISCUSSION THREAD

**Discussion by botzill** [directus/directus#5656] [Archive]
I was wondering if there is a way to format a price filed? Right now I see the integer fields like:

<img width="305" alt="Screen Shot 2021-05-13 at 20 31 04" src="https://user-images.githubusercontent.com/4686040/118163141-51dd0f00-b42a-11eb-8433-907529419192.png">

**Comment by benhaynes** [comment]
We should have a prefix and suffix (or display template) option for the Formatted Value Display. Also, any numbers passed into that Display should add the appropriate commas. Not sure about localization, which might use periods instead of commas.

**Comment by Fusseldieb** [comment]
Sorry to respond to such an old issue, but this would be nice! Some sort of formatter for the display fields. For instance, right now we have a weight field, and each one appears as **10.0000g**, which can become quite confusing. With a simple formatter it could round it, so, at a glance, it would be easier to see.

An idea would be Jinja as a formatter. You'll have tons of options to format of display stuff.

Of couse there are custom extensions, but I think Directus could benefit from it natively.
